omnigibson/simulator.py

```diff
@@ -495,6 +495,14 @@ def play(self):
             with suppress_omni_log(channels=channels):
                 super().play()
 
+            # If we're stopped, take a physics step and update the physics sim view. This must happen BEFORE the
```

Comment:
Curious, why is this necessary to do here instead of where it was before?

Reply:
It's weird -- for whatever reason calling `self._physics_sim_view = omni.physics.tensors.create_simulation_view(self.backend)` causes all the current per-object physics views to break. So they need to be refreshed **after** the global physics sim view is created.